### ISSUE TITLE
Provide a default country dropdown at top right

### ISSUE THREAD

**Issue by kinabalu** [pr3d4t0r/COVIDvu#407]
We may embed a service for identifying based on IP address what country a user is in, in the meantime it would be great to have a dropdown where we could at least select the country of origin and we can save that in a cookie.

For now, would like a selector (perhaps we can remove the Refresh button) which allows them to select their continent? country? @pr3d4t0r @jaryaman @jtdavies ?

**Comment by jaryaman** [comment]
@kinabalu Yeah, being able to store the user's country in a cookie would be awesome. Or just the final state the user leaves the selections...
